### PR TITLE
"host" -> "node", + terminology section

### PR DESCRIPTION
**Pull request by eqvinox**
As noted by Tim on mic for the v6ops draft.

Fixes: #19

### CODE REVIEW

File: draft-collink-6man-pio-pflag.xml [hunk 1]
```diff
@@ -129,16 +129,38 @@ network, so there must be a mechanism for the network to communicate with this t
           all capitals, as shown here.</t>
       </section>
 
+      <section>
+        <name>Terminology</name>
```
[the remark this answers is not in the document]
Comment (eqvinox) [reply draft-collink-6man-pio-pflag.xml:133]: > Is there a reason we need this terminology section?

Hmm, well, no, the problem / reason I added it is…

> […] have slightly different meanings than in the rest of the standards.

… it's kinda this; when we say "router" we mean "the thing sending RAs", and when we say "host"/"node", we mean "receiving RAs, creating an address, performing DHCPv6-PD", and that's close but not exactly how 4862 uses these terms…fv

But I think maybe we can word it like… "This draft is scoped to forming addresses on one particular link. `router` and `node` are used as defined in 4861, keeping this scope in mind."

File: draft-collink-6man-pio-pflag.xml [hunk 2]
```diff
@@ -100,22 +100,22 @@ The flag is used to indicate that the network prefers that hosts acquire global
...
 <t>
-On small networks, scaling to support multiple individual IPv6 addresses is less of a concern, because many home routers support hundreds of neighbor cache entries. On the other hand, address space is more limited compared to the number of hosts connected - the smallest home network might only have /60 prefixes, or even just a single /64.
+On small networks, scaling to support multiple individual IPv6 addresses is less of a concern, because many home routers support hundreds of neighbor cache entries. On the other hand, address space is more limited compared to the number of nodes connected - the smallest home network might only have /60 prefixes, or even just a single /64.
 </t>
 
 <t>
-A host cannot know in advance which address assignment method is most appropriate for the 
-network, so there must be a mechanism for the network to communicate with this to the host.
+A node cannot know in advance which address assignment method is most appropriate for the
+network, so there must be a mechanism for the network to communicate with this to the node.
```
[the remark this answers is not in the document]
Comment (eqvinox) [reply draft-collink-6man-pio-pflag.xml:118]: not related to this PR but ACK

File: draft-collink-6man-pio-pflag.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (eqvinox) [reply draft-collink-6man-pio-pflag.xml]: After thinking about it I just dropped it

**Comment by eqvinox** [comment]
Rebased, dropped terminology section, `with this` → `this`

**Comment by furry13** [comment]
I think we've done everything which needs to be done here..

**Pull request closed without merging**